src/grid/cubic.py

```diff
@@ -0,0 +1,516 @@
+import numpy as np
+from scipy.interpolate import CubicSpline
+
+from grid.basegrid import Grid, OneDGrid
+
+
+class CubicGrid(Grid):
+    r"""
+
+    Attributes
+    ----------
+
+    Methods
+    -------
+
+
+    """
+    def __init__(self, oned_grids):
```

Comment:
Isn't it easier and more clear to describe, if 3 `OneDGrid` are passed instead of a list of them? I.e., what about
```
def __init__(self, oned_x, oned_y, oned_z):
```

Reply:
Resolved in 064daac